<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>npc_gift_option</class>
 <widget class="QDialog" name="npc_gift_option">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>632</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>CrossCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>NPC赠礼</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46,47,48);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="gift_code_output">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="statusTip">
      <string>输出的代码将会显示在这里</string>
     </property>
     <property name="whatsThis">
      <string>输出的代码将会显示在这里</string>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 3px; border: 2px solid #E7BF9F; border-radius: 5px; color: #E7BF9F;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="placeholderText">
      <string>编辑完成请直接退出，重新进入将会清空所有内容。</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="npc_multi_gift">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="statusTip">
        <string>在这里输入赠礼名称</string>
       </property>
       <property name="whatsThis">
        <string>在这里输入赠礼名称</string>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 3px; border: 2px solid #E7BF9F; border-radius: 5px; color: #E7BF9F;</string>
       </property>
       <property name="placeholderText">
        <string>赠礼名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="npc_multi_amount">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="statusTip">
        <string>在这里输入赠礼数量</string>
       </property>
       <property name="whatsThis">
        <string>在这里输入赠礼数量</string>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 3px; border: 2px solid #E7BF9F; border-radius: 5px; color: #E7BF9F;</string>
       </property>
       <property name="placeholderText">
        <string>赠礼数量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="npc_multi_insert_gift">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="statusTip">
        <string>插入当前赠礼</string>
       </property>
       <property name="whatsThis">
        <string>插入当前赠礼</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
background-color: #E7BF9F;
color: rgb(255, 255, 255);
border: 6px;
padding: 5px;</string>
       </property>
       <property name="text">
        <string>插入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="npc_multi_clear_gift">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="statusTip">
        <string>清空整个工作区域</string>
       </property>
       <property name="whatsThis">
        <string>清空整个工作区域</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 5px;
background-color: #E7BF9F;
color: rgb(255, 255, 255);
border: 6px;
padding: 5px;</string>
       </property>
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="npc_gift_mult_submit_final">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 5px;
background-color: #E7BF9F;
color: rgb(255, 255, 255);
border: 6px;
padding: 5px;</string>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
